libs/parallelism/algorithms/tests/unit/container_algorithms/search_range.cpp

```diff
@@ -85,6 +104,8 @@ template <typename IteratorTag>
 void test_search1()
 {
     using namespace hpx::execution;
+    test_search1_without_expolicy(IteratorTag());
```

Comment:
This is a matter of taste. I like to have this simply as an overload of `test_search1` with the execution policy argument missing, but feel free to leave it like this if you prefer that.